### ISSUE TITLE
structured comments do not escape special characters

### ISSUE THREAD

**Issue by GoogleCodeExporter** [franca/franca#54]
```
if characters like "*" or "@" are not accepted within structured comments.

example with errors:
<**
	@author : me@home
**>

<**
	@description : provides the result 2*3
**>

both examples show syntax errors in the Franca editor and can not be parsed for 
transformations.

Both characters should be accepted by the Franca model.

```

Original issue reported on code.google.com by `manfred....@bmw.de` on 24 Jul 2013 at 4:29

**Comment by kbirken** [comment]
See also issues #137 and #173.

**Comment by kbirken** [comment]
Implemented by commit 0480c29 on develop.

The star and '@' characters can be used now in the annotation comment by escaping them with a backslash: \*
When accessing the comment string via the generated EMF API, the escape backslash will be removed.
